README.md

```diff
@@ -1,16 +1,93 @@
+  ...
+}
+```
+It represents the possible input fields.
+As key, you use the feature name, give it a title and decide whether it should be represented as a slider or radio choice buttons. For radio buttons, specify a list of available choices with their correlated number representation in your ML algorithm. For a slider, specify the minimal and maximal value that the user should be able to input.
 
-A few resources to get you started if this is your first Flutter project:
+#### models.json
+In `models.json` you specify, per label, the ML determined coefficients for each feature as well as their means. It should look similar to this:
+```
+{
+  "asthma": {
+    "features": {
+      "age": {
+        "coef": 0.08657563511716311, 
+        "mean": 56.90638617580766
+      },
+      ...
+    }
+  },
+  ...
+}
+```
+First key is the label name, in this case "COPD". inside the "feature" argument you then can specify a list of features that influence this label with the respective coefficients (at key "coeff") and means (at key "mean"). Repeat this for all features and labels.
 
-- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
-- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
+#### labels.conf
+In `assets/labels.conf` you specify all labels that your want to predict for, that also have a representation in `assets/models.json` (!) as a simple list with the key "labels".
+
+```
+{
+  "labels": [
+  	"asthma",
+   ...
+   ]
+}
+```
+
+
+## How to use
+There are two visual representations of the same concept: Bars on startup and Bubbles once you swipe one page to the right.
+Should you have left out some input parameters, the app will assume the means given in `assets/models.json`.
+
+### Bars
+![Alt text](/assets/bars_prototype.png "Bars Prototype")
+On the left side, you can put in your data and press the button in the middle to get the output. You can also change the input while the output is displayed or hide the output bars again by pressing the same button in the middle. To reset to means, press the floating button on the bottom right.
+
+### Bubbles
+![Alt text](/assets/bubbles_prototype.png "Bubbles Prototype")
+In Bubbles Prototype the input fields have a bubble representation. They are located at the top of the screen. The output is also in a circular form in the screen center around a center image. Each output bubble representing one ML prediction label. You can input your data by dragging or clicking an input bubble. A dialog opens and asks you to input your data. By pressing "ok" the change is propagated to the output bubbles and their size changes according to the new probabilities. Additionally there are particles, small bubbles, flying from the input to the influenced output bubble(s) to indicate their correlation.
+
+
+## Contributing
+We are happy if you'd want to contribute to the project. To get into the development, you might want to take some things into account.
+### Naming
```

Comment:
honestly I would remove the whole # Naming section

Reply:
I just thought it might be confusin to people not familiar with ML language.

Reply:
Then they shouldn't use this app :) <3

Reply:
Sure. :D